apps/studio/src/pages/dashboard/edit/index.tsx

```diff
@@ -165,35 +165,38 @@ const EditPage: NextPageWithLayout = () => {
     }
   }
 
+  const { setDrawerState, setPageState, setEditorState } =
+    useEditorDrawerContext()
+  useEffect(() => {
+    setDrawerState({
+      state: 'root',
+    })
+    const blocks = [
+      { text: 'Hero', id: 'hero-123' },
+      { text: 'Content', id: 'content-123' },
+      { text: 'Infopic', id: 'infopic-123' },
+      { text: 'Content', id: 'content-234' },
+    ]
+    setEditorState(blocks)
+    setPageState(blocks)
+  }, [])
```

Comment:
assuming this is temporary for testing? if so, could we add a `TODO` to remove

Reply:
added in [5e92039](https://github.com/opengovsg/isomer/pull/219/commits/5e9203914ea9114d0a8e22d2e772684e85934107)!